src/components/Booklist.js

```diff
@@ -1,14 +1,25 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Books from './Books';
 import FormInput from './FormInput';
+import { fetchBooks } from '../redux/books/books';
 
 const Booklist = () => {
   const Booklist = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchBooks(), []);
+  });
```

Comment:
- [ ] There is mistake while trying to add empty array dependency to the `useEffect`. It should be added after the closing curly brace of the callback function. Take a look :point_down: 

```
useEffect(() => {
    dispatch(fetchBooks());
  }, [dispatch]); 
```
You can add dispatch inside the array to avoid warning from react. Kindly look into it and fix.